service/policy/obligations/obligations.proto

```diff
@@ -0,0 +1,230 @@
+syntax = "proto3";
+
+package policy.obligations;
+
+import "common/common.proto";
+import "policy/objects.proto";
+import "policy/selectors.proto";
+import "google/protobuf/struct.proto";
+
+///
+/// Obligation
+///
+
+// Definitions
+message GetObligationRequest {
+    string id = 1;
+}
+
+message GetObligationResponse {
+    policy.Obligation obligation = 1;
+}
+
+message CreateObligationRequest {
+    // Required
+    string namespace_id = 1;
+    string name = 2;
+    // Optional
+    // Common metadata
+    common.MetadataMutable metadata = 100;
+}
+
+message CreateObligationResponse {
+    policy.Obligation obligation = 1;
+}
+
+message UpdateObligationRequest {
+    // Required
+    string id = 1;
```

Comment:
Should this be oneof `id` or `fqn`?

Reply:
I think `id` is preferable because if you change the name of the obligation, its FQN would be different. 

Also, I think changing the name should be allowed here and safe because the obligation FQNs are not stored in TDF manifests and therefore much more dynamic around access and mutations.